tests/test_estimators.py

```diff
@@ -0,0 +1,77 @@
+from sklearn.datasets import load_iris
+from sklearn.utils.estimator_checks import check_estimator
+
+import geomstats.backend as gs
+assert_allclose = gs.testing.assert_allclose
```

Comment:
It would/should, but we're currently not enforcing flake8 compatibility in the test suite. We probably should though. I'll submit another PR to clean up the test suite, and enable flake8 testing there once https://github.com/geomstats/geomstats/pull/296 is merged, and then refactor this PR on top afterwards.